tests/gateway/test_gateway_handlers.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+import yaml
+
+
+from mlflow.exceptions import MlflowException
+from mlflow.gateway.config import GatewayConfigSingleton
+from mlflow.gateway.handlers import (
+    _load_gateway_config,
+    _save_gateway_config,
+    RouteConfig,
+    _convert_route_config_to_route,
+)
+
+
+@pytest.fixture(autouse=True)
```

Comment:
```suggestion
@pytest.fixture
```

We don't need to use `autouse=True` because this fixture doesn't have any side effects.